### PR TITLE
refactor: Improve Lightweight styling for Material Combo Box

### PR DESCRIPTION
**Pull request by lesterbotello**
﻿GitHub Issue: #1005

## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## Description

Refactor ComboBox styles for Material to follow #894 

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [X] Tested UWP
- [X] Tested iOS
- [X] Tested Android
- [X] Tested WASM
- ~~[ ] Tested MacOS~~
- [X] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

## Internal Issue (If applicable):
#1005

### CODE REVIEW

File: src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml [hunk 1]
```diff
@@ -24,109 +24,109 @@
 		<ResourceDictionary x:Key="Default">
 			
 			<!-- ComboBoxItem Brushes -->
-			<StaticResource x:Key="MaterialComboBoxItemForegroundThemeBrush"
+			<StaticResource x:Key="ComboBoxItemForegroundThemeBrush"
```
[the remark this answers is not in the document]
Comment (lesterbotello) [reply src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml]: Fixed

**Review by Xiaoy312 — APPROVED** [review]

File: src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml [hunk 2]
```diff
@@ -24,109 +24,112 @@
...
 							ResourceKey="OnSurfaceLowBrush" />
 
-			<StaticResource x:Key="MaterialComboBoxFocusedBorderBrush"
+			<!-- TODO: Doesn't directly match Fluent, but there's a ComboBoxFocusedBackgroundThemeBrush instead -->
+			<!-- Is it supposed to be the same style? Use which? -->
+			<StaticResource x:Key="ComboBoxFocusedBorderBrush"
 							ResourceKey="OnBackgroundBrush" />
 
-			<StaticResource x:Key="MaterialComboBoxOpenedBorderBrush"
+			<!-- TODO: No match of this style in Fluent -->
```
Comment (agneszitte) [inline src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:51]: @kazo0 Can we associate these ToDos with an issue + link in the comment?

File: src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:1]: We should be refactoring this to support pretty much all of the resources covererd by the Fluent style here: https://github.com/unoplatform/uno/blob/1ecff68b4c71dfd1f0f7df60fbbed9dd27ac1952/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/ComboBox_themeresources.xaml#L19-L123

Comment (lesterbotello) [reply src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:1]: Fixed. Most of the brushes from Fluent have been ported, Some other which I found were not being actually used in Fluent were skipped

File: src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml [hunk 3]
```diff
@@ -22,121 +22,464 @@
 	<ResourceDictionary.ThemeDictionaries>
 
 		<ResourceDictionary x:Key="Default">
```
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:24]: Looks good! Only thing is the formatting, if you run Xaml Styler it should fix this, unless it's overridden in your XamlStyler settings but the x:Key and the ResourceKey should be on the same line for the StaticResource.

So it should look organized by property like we do in other styles like [here](https://github.com/unoplatform/Uno.Themes/blob/d5d803370cc99e11e3e895992a220b95abec477a/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L27-L50)

**Review by kazo0 — APPROVED** [review]

File: src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml (hunk 3 above)
Comment (lesterbotello) [reply src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:24]: Fixed

**Pull request merged**